cmd/init.go

```diff
@@ -97,6 +99,15 @@ to quickly create a Cobra application.`,
 		}
 		log.Println("region:", region)
 
+		profile, _ := cmd.Flags().GetString("profile")
+		viper.Set("aws.profile", profile)
+		// propagate it to local environment
```

Comment:
Should we check for error here?
profile, _ := cmd.Flags().GetString("profile")

```golang
profile, err := cmd.Flags().GetString("profile")
		if err != nil {
			log.Println(err)
		}
```

Reply:
its always nice to be ready the day it will go wrong xD